ts/Dashboards/Board.ts

```diff
@@ -413,37 +503,105 @@ namespace Board {
      *
      * */
 
+    /**
+     * Options to configure the board.
+     **/
     export interface Options {
+        /**
+         * Options for the GUI. Allows to define graphical elements and its
+         * layout.
+         **/
         gui?: GUIOptions;
+        /**
+         * Options for the edit mode. Can be used to enable the edit mode and
+         * define al things related to it like the context menu.
+         **/
         editMode?: EditMode.Options;
+        /**
+         * List of components to add to the board.
+         **/
         components?: Array<Bindings.ComponentOptions>;
+        /**
+         * General options for the components.
+         **/
         componentOptions?: Partial<Bindings.ComponentOptions>;
+        /**
+         * A list of serialized layouts to add to the board.
+         **/
         layoutsJSON?: Array<Layout.JSON>;
-        respoBreakpoints?: RespoBreakpoints;
+        /**
+         * Responsive breakpoints for the board- small, medium and large.
+         **/
+        responsiveBreakpoints?: ResponsiveBreakpoints;
     }
 
+    /**
+     * Serialized options to configure the board.
+     **/
     export interface OptionsJSON extends JSON.Object {
+        /**
+         * Id of the container to which the board is added.
+         **/
         containerId: string;
+        /**
+         * An array of serialized layouts to add to the board.
+         **/
         layouts: Array<Layout.JSON>;
+        /**
+         * Whether the GUI is enabled or not.
+         **/
         guiEnabled?: boolean;
+        /**
+         * General options for the components.
+         **/
         componentOptions?: Partial<Bindings.ComponentOptions>;
-        respoBreakpoints?: RespoBreakpoints;
+        /**
+         * Responsive breakpoints for the board- small, medium and large.
+         **/
+        responsiveBreakpoints?: ResponsiveBreakpoints;
     }
 
-    export interface RespoBreakpoints extends JSON.Object {
+    /**
+     * Responsive breakpoints for the board- small, medium and large.
+     **/
+    export interface ResponsiveBreakpoints extends JSON.Object {
+        /**
+         * Value in px to test the dashboard is in small mode.
+         **/
         small: number;
+        /**
+         * Value in px to test the dashboard is in medium mode.
+         **/
         medium: number;
+        /**
+         * Value in px to test the dashboard is in large mode.
+         **/
         large: number;
     }
 
     export interface GUIOptions {
+        /**
+         * Whether the GUI is enabled or not.
```

Comment:
Does it make sense to have that `enable` here? When can we use this property?

IMO it is redundant, what is your opinion?

Reply:
I agree. It seems to be redundant. Maybe it is supposed to control visibility?

Reply:
I agree. It seems to be redundant. Maybe it is supposed to control visibility?

Reply:
If we keep it, it should be true by default.